devtools-html/debugger.html
# WebAssembly text generation on debugger side

Associated Issue: n/a

### Summary of Changes

* adds code generation for editor
* better source-to-editor text line mapping

### Test Plan

Example test plan (for original source):

1. Open https://devtools-html.github.io/debugger-examples/examples/wasm/fib/fib.index.html
2. Open debugger and refresh fib.index.html page (the latter is to enable wasm debugging)
3. Find/open "fib.c"
4. Set breakpoint at line 4
5. Refresh refresh fib.index.html page

Alternative test plan (for wasm):

3a. Find/open "wasm:https://devtools-html.github.io/debugger-examples/examples/wasm/fib/fib.index.html:d65901927dbe8357"
4a. Set breakpoint at the beginnig of $func5 @0000027D
5a. Refresh refresh fib.index.html page

Expected results: stop at set breakpoint(s).

Some other defects found:
- [ ] unable to set breakpoints in step 4a
- [ ] when trying to step through in original source (after step 5), it take multiple step commands to pass to the next line.
